Index: build.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\n\n#pyinstaller -F -w -i python_tool.icns --name \"Updater\" update.py\npython3 -m nuitka --standalone --macos-create-app-bundle --remove-output --enable-plugins=tk-inter --macos-app-name=Python_tool --macos-app-version=1.1.0 --show-progress --output-dir=dist --macos-app-icon=python_tool.icns python_tool.py\n#python -m nuitka --clang --remove-output --show-progress --standalone --output-dir=udist --macos-create-app-bundle --macos-app-name=Updater --enable-plugins=tk-inter --macos-app-version=1.0.0 --macos-app-icon=python_tool.icns update.py\n\n
===================================================================
diff --git a/build.sh b/build.sh
--- a/build.sh	(revision 79c0db9385cf136daa748d11d38e003c94694b08)
+++ b/build.sh	(date 1734772566480)
@@ -1,6 +1,6 @@
 
 
-#pyinstaller -F -w -i python_tool.icns --name "Updater" update.py
+
 python3 -m nuitka --standalone --macos-create-app-bundle --remove-output --enable-plugins=tk-inter --macos-app-name=Python_tool --macos-app-version=1.1.0 --show-progress --output-dir=dist --macos-app-icon=python_tool.icns python_tool.py
 #python -m nuitka --clang --remove-output --show-progress --standalone --output-dir=udist --macos-create-app-bundle --macos-app-name=Updater --enable-plugins=tk-inter --macos-app-version=1.0.0 --macos-app-icon=python_tool.icns update.py
 
